src/core/components/courses/ProfileFollowUp.vue

```diff
@@ -88,6 +110,12 @@ export default {
   },
   computed: {
     ...mapGetters({ company: 'main/getCompany' }),
+    traineesOnGoingCount () {
+      return this.learners.filter(l => l.progress !== 1).length;
+    },
+    traineesFinnishedCount () {
+      return this.learners.filter(l => l.progress === 1).length;
+    },
```

Comment:
Est-ce qu'on peut directement le calculer comme ça : 
`this.learners.length - this.traineesOnGoingCount` ?

Reply:
ça évite de parcourir deux fois la liste à chaque changement